src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxNormalizerTests.cs

```diff
@@ -942,6 +942,16 @@ public void TestNormalizeBlockAnonymousFunctions(string actual, string expected)
             TestNormalizeStatement(actual, expected);
         }
 
+        [Fact(Skip = "PROTOTYPE")]
```

Comment:
can we link this to an actual tracking bug?

Reply:
No need. This is going into a feature branch and all PROTOTYPE markers need to be addressed (or converted to issues) before merging the feature.